js/admin.block-post.js

```diff
@@ -108,6 +113,12 @@
 		}
 	};
 
+	let blockFieldInit = function() {
```

Comment:
At some stage we need to get this into a build process so that we can start using ES6+ syntax, but this works for now.